### PR TITLE
Closes #634.

### LINKED ISSUE #634: [Typography] First CSS details for each of the pairings is too close to the h6 border bottom
On Chrome, the css details are too close to the h6 border bottom and could use some more space and should top-align with the Display 1.
image

### PR DESCRIPTION
**Pull request by Taitrnator**

### CODE REVIEW

File: assets/css/styleguide.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+Error: Invalid CSS after "@import 'all'": expected "{", was ";"
```
Comment (msecret) [inline assets/css/styleguide.css]: I think theres a problem in the sass which is causing an error here.

File: assets/css/styleguide.scss [hunk 2]
```diff
@@ -77,9 +77,9 @@ $site-top:           124px;
       float: right;
       display: none;
       margin-top: -5px;
-      
-      li { 
-        display: inline; 
+
+      li {
```
Comment (msecret) [inline assets/css/styleguide.scss]: I can't find the matching brace for this. It's either missing the closing brace or the alignment is off.

File: assets/css/styleguide.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/scss/parser.rb:117:in `stylesheet'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/scss/parser.rb:42:in `parse'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/engine.rb:403:in `_to_tree'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/engine.rb:278:in `render'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/exec/sass_scss.rb:418:in `run'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/exec/sass_scss.rb:63:in `process_result'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/exec/base.rb:52:in `parse'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/lib/sass/exec/base.rb:19:in `parse!'
+/Applications/CodeKit.app/Contents/Resources/engines/scss/bin/scss:13:in `<main>'
+*/
```
Comment (maya) [inline assets/css/styleguide.css]: I don't think we want to add this Codekit reference.

**Comment by maya** [comment]
I see you adde a `margin-bottom` on line 830. We don't want to add a margin bottom for all h6 alt headings, it's just that specific instance that has an issue. For those reasons, closing this PR.

Also if you could add a description in future PR's, we'd be able to understand it better and that would be appreciated.

**Pull request closed without merging**